Ginger/GingerCoreNET/GingerCoreNET.csproj

```diff
@@ -31,10 +31,16 @@
   <ItemGroup>
     <Compile Remove="DataBaseAbstarct\**" />
     <Compile Remove="GeneralLib\JSON\**" />
+    <Compile Remove="NewFolder1\**" />
+    <Compile Remove="NewFolder\**" />
     <EmbeddedResource Remove="DataBaseAbstarct\**" />
     <EmbeddedResource Remove="GeneralLib\JSON\**" />
+    <EmbeddedResource Remove="NewFolder1\**" />
+    <EmbeddedResource Remove="NewFolder\**" />
     <None Remove="DataBaseAbstarct\**" />
     <None Remove="GeneralLib\JSON\**" />
+    <None Remove="NewFolder1\**" />
```

Comment:
Check and remove this lines if not needed

Reply:
done